docs/examples/basic-requirements.md

```diff
@@ -14,7 +14,7 @@ As root (either ssh directly to root, local root console, or by using "sudo su -
 ```bash
 cd ~
 curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
-curl -LO https://github.com/kubernetes/kops/releases/download/1.7.0/kops-linux-amd64
+curl -LO https://github.com/kubernetes/kops/releases/download/1.7.1/kops-linux-amd64

```

Comment:
Can we do that curl call for the latest release?  I know there is a curl command for that.

Reply:
@chrislovecnm I'll look at that, although "latest" right now is technically `1.8.0-beta.1`, should that me move to `pre-release`?

Reply:
Also @chrislovecnm google seems to say it's only possible if we do something like this
```
curl -LO https://github.com/kubernetes/kops/releases/download/$(curl -s https://api.github.com/repos/kubernetes/kops/releases/latest | grep tag_name | cut -d '"' -f 4)/kops-linux-amd64
```

Let me know if that's acceptable!